test/test_files/tinysnb/function/list.test

```diff
@@ -827,3 +827,163 @@ True
 -QUERY Return list_sum([1.1, 2.2, 3.3, NULL]);
 ---- 1
 6.600000
+
```

Comment:
Instead of simply testing on list literals, can you use tinySnb dataset to test the string functions?
You can find tinySnb dataset under: `dataset/tinysnb`